src/Presenters/BulmaPresenter.php

```diff
@@ -0,0 +1,428 @@
+    protected function isButton()
+    {
+        return $this->element instanceof Button;
+    }
+
+    /**
+     * @return bool
+     */
+    protected function isSelect()
+    {
+        return $this->element instanceof Select && !$this->element->multiple;
+    }
+
+    /**
+     * @return bool
+     */
+    protected function isList()
+    {
+        return $this->element instanceof Checklist;
+    }
+
+    /**
+     * @param Open $open
+     * @return string
+     */
+    public function renderOpeningTag(Open $open)
+    {
+        if (isset($this->style['form']) && !$this->forcedClass) {
+            $open->addClass($this->style['form']);
+        }
+
+        $result = '<form' . $this->renderAttributes($open->attributes) . '>';
+        return $result;
+    }
+
+    /**
+     * @param Close $close
+     * @return string
+     */
+    public function renderClosingTag(Close $close)
+    {
+        $result = '</form>';
+        return $result;
+    }
+
+    /**
+     * @param Button $button
+     * @return string
+     */
+    public function renderButton(Button $button)
+    {
+        return '<button' . $this->renderAttributes($button->attributes) . '>' . $button->text . '</button>';
+    }
+
+    /**
+     * @return string
+     */
+    public function renderLabel()
+    {
+        $result = '';
+
+        if ($this->isHorizontal()) {
+            $normalClass = !$this->element instanceof Checklist ? ' is-normal' : '';
+            $result .= '<div class="field-label' . $normalClass . '">';
+        }
+
+        if ($this->label) {
+            $class = $this->style['label'];
+            if ($this->error) {
+                $class .= ' has-text-danger';
+            }
+
+            $result .= '<label class="' . $class . '" for="' . $this->attributes['id'] . '">' . $this->label . '</label>';
+        }
+
+        if ($this->isHorizontal()) {
+            $result .= '</div>';
+        }
+
+        return $result;
+    }
+
+    public function addElementClasses() {
+
+        if ($this->element instanceof Input) {
+            $this->element->addClass('input');
+        } elseif ($this->element instanceof Button) {
+            $this->element->addClass('button');
+        } elseif ($this->element instanceof TextArea) {
+            $this->element->addClass('textarea');
+        } elseif ($this->element instanceof Radio) {
+            $this->element->addClass('radio');
+        } elseif ($this->element instanceof Checkbox) {
+            $this->element->addClass('checkbox');
+        } elseif ($this->element instanceof Checklist) {
+            if ($this->element->multiple) {
+                $this->element->addClass('checkbox');
+            } else {
+                $this->element->addClass('radio');
+            }
+        } elseif ($this->element instanceof Select) {
+            $this->element->addClass('select');
+        } elseif ($this->element instanceof TextArea) {
+            $this->element->addClass('textarea');
+        }
```

Comment:
Bi tukaj mogoče lahko naredil $class = strtolower(class_basename($this->element)) in samo Checklist opcijo obravnavaš posebej?